docs/guide/workspace.md

```diff
@@ -14,9 +14,19 @@ Vitest provides a way to define multiple project configurations within a single
 
 ## Defining a Workspace
 
-A workspace must include a `vitest.workspace` or `vitest.projects` file in its root directory (located in the same folder as your root configuration file or working directory if it doesn't exist). Note that `projects` is just an alias and does not change the behavior or semantics of this feature. Vitest supports `ts`, `js`, and `json` extensions for this file.
+Since Vitest 3, you can define a workspace in you root [config](/config). In this case, Vitest will ignore the `vitest.workspace` file in the root, if one exists.
```

Comment:
```suggestion
Since Vitest 3, you can define a workspace in your root [config](/config). In this case, Vitest will ignore the `vitest.workspace` file in the root, if one exists.
```